### PR TITLE
fix: handle unknown language errors in code block rendering [KHCP-20402]

### PR DESCRIPTION
**Pull request by arashsheyda**
# Summary

ticket: https://konghq.atlassian.net/browse/KHCP-20402

this PR handles unknown language errors in code block rendering

### CODE REVIEW

File: src/composables/useMarkdownIt.ts [hunk 1]
```diff
@@ -123,7 +123,17 @@ export default function useMarkdownIt() {
...
+      // Shiki throws when a fenced code block specifies an unknown or unloaded language
+      // (e.g. ```this is will break). Catch the error and fall back to a plain <pre><code>
+      // block so that rendering continues for the rest of the document.
+      let originalContent: string
+      try {
+        originalContent = defaultCodeblockRenderer(tokens, idx, options, env, self)
+      } catch {
+        const escaped = md.value.utils.escapeHtml(tokens[idx]?.content)
+        originalContent = `<pre><code>${escaped}</code></pre>`
+      }
```
Comment (adamdehaven) [inline src/composables/useMarkdownIt.ts:136]: If this fixes something that is broken, please add tests (positive path, and negative path) in the PR

Comment (arashsheyda) [reply src/composables/useMarkdownIt.ts:136]: updated

**Comment by adamdehaven** [comment]
getting an error running the dev server:

```
failed to load config markdown/vite.config.ts
error when starting dev server:
TypeError: localStorage.getItem is not a function
```

**Comment by arashsheyda** [comment]
> getting an error running the dev server:

can't reproduce it 🤔

**Review by adamdehaven — APPROVED** [review]
lgtm
